ibm-mq-metrics/model/metrics.yaml

```
@@ -0,0 +1,611 @@
+      - ref: ibm.mq.job.name
+        requirement_level: required
+      - ref: ibm.mq.channel.start.time
+        requirement_level: required
+  - id: ibm.mq.message.count
+    type: metric
+    metric_name: ibm.mq.message.count
+    stability: development
+    brief: "Message count"
+    unit: "{message}"
+    instrument: gauge
+    attributes:
+      - ref: ibm.mq.channel.name
+        requirement_level: required
+      - ref: ibm.mq.channel.type
+        requirement_level: required
+      - ref: ibm.mq.queue.manager
+        requirement_level: required
+      - ref: ibm.mq.job.name
+        requirement_level: required
+      - ref: ibm.mq.channel.start.time
+        requirement_level: required
+  - id: ibm.mq.open.input.count
+    type: metric
+    metric_name: ibm.mq.open.input.count
+    stability: development
+    brief: "Count of applications sending messages to the queue"
+    instrument: gauge
+    unit: "{application}"
+    attributes:
+      - ref: ibm.mq.queue.manager
+        requirement_level: required
+      - ref: messaging.destination.name
+        requirement_level: required
+      - ref: ibm.mq.queue.type
+        requirement_level: required
+  - id: ibm.mq.open.output.count
+    type: metric
+    metric_name: ibm.mq.open.output.count
+    stability: development
+    brief: "Count of applications consuming messages from the queue"
+    instrument: gauge
+    unit: "{application}"
+    attributes:
+      - ref: ibm.mq.queue.manager
+        requirement_level: required
+      - ref: messaging.destination.name
+        requirement_level: required
+      - ref: ibm.mq.queue.type
+        requirement_level: required
+  - id: ibm.mq.high.queue.depth
+    type: metric
+    metric_name: ibm.mq.high.queue.depth
+    stability: development
+    brief: "The current high queue depth"
+    instrument: gauge
+    unit: "{percent}"
+    attributes:
+      - ref: ibm.mq.queue.manager
+        requirement_level: required
+      - ref: messaging.destination.name
+        requirement_level: required
+      - ref: ibm.mq.queue.type
+        requirement_level: required
+  - id: ibm.mq.service.interval
+    type: metric
+    metric_name: ibm.mq.service.interval
+    stability: development
+    brief: "The queue service interval"
+    instrument: gauge
+    unit: "{percent}"
+    attributes:
+      - ref: ibm.mq.queue.manager
+        requirement_level: required
+      - ref: messaging.destination.name
+        requirement_level: required
+      - ref: ibm.mq.queue.type
+        requirement_level: required
+  - id: ibm.mq.queue.depth.full.event
+    type: metric
+    metric_name: ibm.mq.queue.depth.full.event
+    stability: development
+    brief: "The number of full queue events"
+    instrument: counter
+    unit: "{event}"
+    attributes:
+      - ref: ibm.mq.queue.manager
+        requirement_level: required
+      - ref: messaging.destination.name
+        requirement_level: required
+  - id: ibm.mq..queue.depth.high.event
```

how did weaver not catch this?
```suggestion
  - id: ibm.mq.queue.depth.high.event
```